### PR TITLE
Vector Fitting: bugfix for data including dc samples

### PR DESCRIPTION
**Pull request by Vinc0110**
Vector Fitting crashes if the fitted data includes a dc sample.

The first starting pole $p_0$ is always positioned at $\omega_0=2\pi f[0]$, which is $0$ in such cases. The respective coefficient then becomes `inf` or `NaN` and causes the crash:
$\frac{1}{j \pi f_0 - p_0} = \frac{1}{0-0}$.

This fix places $p_0$ at $\omega_0=2\pi f[1] / 1000$, so the coefficient is less problematic:
$\frac{1}{j \pi f_0 - p_0} = \frac{1}{0-\omega_1 / 1000}$.

The factor 1/1000 is chosen arbitrarily and works well in my tests. There might be a better way to do it, but let's go with this for now.

### CODE REVIEW

**Comment by FranzForstmayr** [comment]
Could you add a testcase please? 
Great fix anyway :)

**Comment by Vinc0110** [comment]
Done. 
Are the tests not running on the updated code of this PR? They don't fail when I run them locally...

File: skrf/tests/test_vectorfitting.py [hunk 1]
```diff
@@ -44,6 +44,14 @@ def test_no_convergence(self):
             vf.vector_fit(n_poles_real=0, n_poles_cmplx=5, fit_proportional=False, fit_constant=True)
             self.assertEqual(warning[-1].category, RuntimeWarning)
 
+    def test_dc(self):
+        # perform the fit on data including a dc sample (0 Hz)
+        nw = skrf.Network('./cst_example_4ports.s4p')
```
Comment (FranzForstmayr) [inline skrf/tests/test_vectorfitting.py:49]: The s4p files are usually not in the root folder, maybe that's the issue?

Comment (Vinc0110) [reply skrf/tests/test_vectorfitting.py]: That was it. Thanks!

**Review by jhillairet — APPROVED** [review]

**Comment by jhillairet** [comment]
ok for merging

**Pull request merged**